zipkin-lens/src/actions/dependencies-action.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { DependenciesAction, ActionTypes } from '../types/action-types';
+import Dependencies from '../types/Dependencies';
+import AppThunk from '../types/AppThunk';
+import * as api from '../constants/api';
+
+export const loadDependenciesRequest = (): DependenciesAction => ({
```

Comment:
I guess in typescript it's pretty common to leave out return type for most functions, especially lambdas without logic, and let it infer.